services/pulumi/crds.yaml

```diff
@@ -0,0 +1,26 @@
+helm-crds:
+  - chart: traefik/traefik
+    version: 28.3.0
```

Comment:
While handling Helm charts you might have to consider that the chart might not be in the official repos. For instance `traefik/traefik` is not on the default repo and must be added with: 

`helm repo add traefik https://traefik.github.io/charts` 

In your case, it worked because you've already added or the dev script already added those for you, it might not work on situations that don't have those additional repos. 
So I suggest adding a `repo` optional field to each chart, wdyt?

Reply:
Yeah, I was thinking about that a few minutes ago. I think we can do it, no problem!